package.json

```diff
@@ -31,6 +17,29 @@
     "node": "^10",
     "yarn": "^1.16.0"
   },
+  "scripts": {
+    "build": "tsc --build",
+    "dev": "concurrently --kill-others \"yarn watch\" \"yarn serve\"",
+    "deploy": "yarn install && yarn demolish && yarn build",
+    "demolish": "rimraf dist",
+    "serve": "nodemon --watch dist --watch config --delay 4 dist/_main.js",
+    "start": "node dist/_main.js",
+    "lint": "yarn lint-tslint && yarn lint-prettier",
+    "lint-tslint": "tslint --config tslint.json --format stylish --project tsconfig.json",
+    "lint-prettier": "prettier --check \"*.{js,json,md,yml}\" \"{.github,@types,config,data,src}/**/*.{js,json,md,ts,yml}\"",
```

Comment:
I don't like this name, it's not a linter with the traditional  meaning of the term and it may become confusing

Reply:
What name would you suggest?

Reply:
Ok, I only left `lint` for format checking and `format` for reformatting.